src/NServiceBus.Extensions.Logging.NLog.AcceptanceTests/When_using_nlog_extension_logger.cs

```diff
@@ -17,7 +17,7 @@ public async Task Should_log_nsb_logs()
             config.AddRuleForAllLevels(memoryTarget);
             LogManager.Configuration = config;
             
-            NsbLogManager.UseFactory(new ExtensionsLoggerFactory(new NLogLoggerFactory()));
+            NsbLogManager.Use<ExtensionLogging<NLogLoggerFactory>>();
```

Comment:
Is the double generics really needed? Is it possible to get it down to

`NsbLogManager.Use<NLogLoggerFactory>();`

Reply:
Yes. The Use<T> is in the core and it accepts a `Definition`. If we want to create something generic that'd accept all possible loggers out there, it is by definition something _generic_.

If this gets merged in the core, it can be simplified.